docs/send-data/kubernetes/troubleshoot-collection.md

```diff
@@ -209,6 +209,101 @@ otellogs:
 
 For more information, see [Setting different resources on different nodes for logs collector](/docs/send-data/kubernetes/best-practices/#setting-different-resources-on-different-nodes-for-logs-collector).
 
+### Check logs body
+
+You can print logs on stdout of logs collector and logs metadata, and validate if they are correct. It may happen that logs are ingested, but with different metadata than you expect them.
+
+In order to print them on stdout, two steps are required:
+
+1. Disable ingesting logs from logs related pods. This is required to prevent logs ingest spike. Add the following configuration to `user-values.yaml`:
+
+   ```yaml
+   debug:
+     logs:
+       metadata:
+         stopLogsIngestion: true
+       collector:
+         stopLogsIngestion: true
+   ```
+
+   Then update your collection and wait for all logs collector pods to be redeployed.
+
+2. Enable printing logs on stdout for logs related pods, by adding the following to `user-values.yaml`:
+
+   ```yaml
+   debug:
+     logs:
+       metadata:
+         print: true
+         stopLogsIngestion: true
+       collector:
+         print: true
+         stopLogsIngestion: true
+   ```
+
+3. To revert changes, please perform first step as it is, and after configuration has been propagated to all pods, you can remove all configuration added in this section from the `user-values.yaml`.
+
```

Comment:
```suggestion
1. Disable ingesting logs from logs related pods. This is required to prevent logs ingest spike.
   - Add the following configuration to `user-values.yaml`:
     ```yaml
     debug:
       logs:
         metadata:
           stopLogsIngestion: true
         collector:
           stopLogsIngestion: true
     ```
   - Then, update your collection and wait for all logs collector pods to be redeployed.
2. Enable printing logs on stdout for logs related pods by adding the following to `user-values.yaml`:
   ```yaml
      debug:
        logs:
          metadata:
            print: true
            stopLogsIngestion: true
          collector:
            print: true
            stopLogsIngestion: true
      ```
3. To revert changes, please perform first step as it is, and after configuration has been propagated to all pods, you can remove all configuration added in this section from the `user-values.yaml`.
```